/**:
  ros__parameters:
    stop_line:
      stop_margin: 0.0
      stop_duration_sec: 1.0
      hold_stop_margin_distance: 2.0
